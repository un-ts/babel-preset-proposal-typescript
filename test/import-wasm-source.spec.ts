import { execute } from './helpers.js'
import _ from './import-wasm-source.js'

const proposal = 'import-wasm-source'

test(proposal, () => {
  expect(() => execute(proposal)).toThrowErrorMatchingSnapshot()
  expect(_()).toBeDefined()
})
