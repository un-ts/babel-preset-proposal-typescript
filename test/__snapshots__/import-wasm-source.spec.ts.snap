// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`import-wasm-source 1`] = `"from is not defined"`;
